doc/stdlib/sort.rst

```diff
@@ -119,39 +119,45 @@ Sorting Functions
-   keyword lets you provide a function that will be applied to each element before
-   comparison; the ``lt`` keyword allows providing a custom "less than" function; use
-   ``rev=true`` to reverse the sorting order. These options are independent and can be
-   used together in all possible combinations: if both ``by`` and ``lt`` are specified,
-   the ``lt`` function is applied to the result of the ``by`` function; ``rev=true``
-   reverses whatever ordering specified via the ``by`` and ``lt`` keywords.
+   .. Docstring generated from Julia source
+   .. code-block:: julia
+       sort!(v, [alg=<algorithm>,] [by=<transform>,] [lt=<comparison>,] [rev=false])
+
+   Sort the vector ``v`` in place. ``QuickSort`` is used by default for numeric arrays while ``MergeSort`` is used for other arrays. You can specify an algorithm to use via the ``alg`` keyword (see Sorting Algorithms_ for available algorithms). The ``by`` keyword lets you provide a function that will be applied to each element before comparison; the ``lt`` keyword allows providing a custom "less than" function; use ``rev=true`` to reverse the sorting order. These options are independent and can be used together in all possible combinations: if both ``by`` and ``lt`` are specified, the ``lt`` function is applied to the result of the ``by`` function; ``rev=true`` reverses whatever ordering specified via the ``by`` and ``lt`` keywords.
 
 .. function:: sort(v, [alg=<algorithm>,] [by=<transform>,] [lt=<comparison>,] [rev=false])
 
+   .. Docstring generated from Julia source
+   .. code-block:: julia
+       sort(v, [alg=<algorithm>,] [by=<transform>,] [lt=<comparison>,] [rev=false])
+
    Variant of ``sort!`` that returns a sorted copy of ``v`` leaving ``v`` itself unmodified.
 
-   ::
-              sort(A, dim, [alg=<algorithm>,] [by=<transform>,] [lt=<comparison>,] [rev=false])
+   .. code-block:: julia
+       sort(A, dim, [alg=<algorithm>,] [by=<transform>,] [lt=<comparison>,] [rev=false])
 
    Sort a multidimensional array ``A`` along the given dimension.
 
 .. function:: sort(A, dim, [alg=<algorithm>,] [by=<transform>,] [lt=<comparison>,] [rev=false])
 
-   ::
-              sort(v, [alg=<algorithm>,] [by=<transform>,] [lt=<comparison>,] [rev=false])
+   .. Docstring generated from Julia source
+   .. code-block:: julia
+       sort(v, [alg=<algorithm>,] [by=<transform>,] [lt=<comparison>,] [rev=false])
 
    Variant of ``sort!`` that returns a sorted copy of ``v`` leaving ``v`` itself unmodified.
 
-   ::
-              sort(A, dim, [alg=<algorithm>,] [by=<transform>,] [lt=<comparison>,] [rev=false])
+   .. code-block:: julia
+       sort(A, dim, [alg=<algorithm>,] [by=<transform>,] [lt=<comparison>,] [rev=false])
 
    Sort a multidimensional array ``A`` along the given dimension.
 
 .. function:: sortperm(v, [alg=<algorithm>,] [by=<transform>,] [lt=<comparison>,] [rev=false])
 
+   .. Docstring generated from Julia source
+   ::
```

Comment:
This isn't related to the markdown conversion since the `sortperm` and co docstrings haven't been converted to markdown